npm/README.md

```diff
@@ -0,0 +1,10 @@
+# @unsplash/intlc
+
+_This package is public for convenience but has peer requirements over `fp-ts` and all so it wouldn't necessarily make sense for all kinds of codebases_
```

Comment:
Could we raise an issue to remove this peer dependency?

Reply:
I guess it depends if we want to open source the npm package or not.

Reply:
Technically it's in an open source repo published publically to npm. :wink: